class/class.2.1/hello.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <string>
+
+const std::string FILENAME = "user.txt";
+
+class UserStats {
+public:
+    void load() {
+        std::ifstream infile(FILENAME);
+        if (!infile) {
+            std::cerr << "Failed to open file for reading.\n";
```

Comment:
std::println(stderr, "....");
Also it is better to return some bool flag or optional or std::expected to provide the result of call. Therefore printing logs will be moved to the side of client of the class and the proper message or error handling can be provided - for example. retry or exit from program.